tpgtools/resource.go

```diff
@@ -203,6 +212,14 @@ func (r Resource) DCLPackage() string {
 	return r.productMetadata.DCLPackage()
 }
 
+// IsAlternateLocation returns whether this resource is an additional version
+// of the DCL resource with a different location type. All references in samples
+// to a resource with an alternate location will point to the main version.
+func (r Resource) IsAlternateLocation() bool {
+	// For now, we consider non-regional resources to be alternate.
```

Comment:
I'd extend this comment to say "non-locational resources have the empty string for their location."

Reply:
Done